crates/biome_js_analyze/src/assists/correctness/organize_imports.rs

```diff
@@ -376,22 +329,12 @@ impl From<JsImport> for ImportNode {
 impl ImportNode {
     /// Returns `true` if the named import specifiers of this import node are sorted
     fn is_sorted(&self) -> bool {
-        let mut iter = self
-            .specifiers
-            .values()
-            .map(|(node, _)| node.syntax().text_range().start());
-
-        let Some(mut last) = iter.next() else {
-            return true;
-        };
-
-        iter.all(|value| {
-            if value < last {
-                return false;
-            }
-
-            last = value;
-            true
+        let mut previous = TextSize::from(0);
```

Comment:
nit: I think the previous implementation was better because `previous`/`last` was initialised with a value that exists.